build.sbt

```diff
@@ -114,11 +120,11 @@ lazy val backend = (project in file("backend"))
       "com.typesafe.akka" %% "akka-cluster-typed" % "2.6.20", // should match what we get transitively from Play
       "org.neo4j.driver" % "neo4j-java-driver" % "1.6.3",
       "com.pff" % "java-libpst" % "0.9.3",
+      // TODO TIKA: check the below
       // NOTE: When you update tika you need to check if there are any updates required to be made to the
       // conf/org/apache/tika/mimecustom-mimetypes.xml file
-      "org.apache.tika" % "tika-parsers" % "1.28.5" exclude("javax.ws.rs", "javax.ws.rs-api"),
-      // Daft workaround due to https://github.com/sbt/sbt/issues/3618#issuecomment-454528463
-      "jakarta.ws.rs" % "jakarta.ws.rs-api" % "2.1.5",
+      "org.apache.tika" % "tika-parsers-standard-package" % "2.7.0",
```

Comment:
I think this is the reason we don't need the daft workaround anymore:

https://github.com/DataDog/datadog-api-client-java/issues/481